ios/FluentUI/Core/FluentUIStyle.generated.swift

```diff
@@ -643,6 +643,93 @@ open class FluentUIStyle: NSObject {
 		}
 
 
+		// MARK: - Notification
+		open var Notification: NotificationAppearanceProxy {
+			return NotificationAppearanceProxy(proxy: mainProxy)
+		}
+		open class NotificationAppearanceProxy {
+			public let mainProxy: () -> FluentUIStyle
+			public init(proxy: @escaping () -> FluentUIStyle) {
+				self.mainProxy = proxy
+			}
+
+			// MARK: - dangerToastBackground 
+			open var dangerToastBackground: UIColor {
+				return UIColor(light: UIColor(red: 0.99215686, green: 0.9647059, blue: 0.9647059, alpha: 1.0), lightHighContrast: nil, lightElevated: nil, lightElevatedHighContrast: nil, dark: UIColor(red: 0.24705882, green: 0.0627451, blue: 0.06666667, alpha: 1.0), darkHighContrast: nil, darkElevated: nil, darkElevatedHighContrast: nil)
```

Comment:
something isn't generated write. we should be using fluentcolors/darkredtin40 for example rather than these floating numbers which will not be true to the hex value...